buildscripts/build.sh

```diff
@@ -108,10 +103,7 @@ if [ $GOOS = "windows" ]; then
 fi
 
 env GOOS=$GOOS GOARCH=$GOARCH go build ${BUILD_TAG} -ldflags \
-    "-X github.com/openebs/maya/pkg/version.GitCommit=${GIT_COMMIT} \
-    -X main.CtlName='${CTLNAME}' \
-    -X github.com/openebs/maya/pkg/version.Version=${VERSION} \
-    -X github.com/openebs/dynamic-nfs-provisioner/provisioner.NFSServerDefaultImage=${NFSSERVERIMG}" \
```

Comment:
@mynktl Can the version and git commit be kept like [this](https://github.com/openebs/node-disk-manager/blob/master/build/build.sh#L86). This will help to log initially,   what version and which commit we are running.

Reply:
yes, we can add it. I'll take this in separate PR since this will increase the scope of this PR.

Reply:
ok.